docs/detections/alerts-view-details.asciidoc

```diff
@@ -31,6 +31,7 @@ image::images/alert-details-flyout-right-panel.png[Right panel of the alert deta
 From the right panel, you can also:
 
 * Click **Expand details** to open the <<left-panel,left panel>>, which shows more information about sections in the right panel.
+* Click the history icon (image:images/history-icon.png[History icon,15,15]) to display a list of places that you visited from the alert's details flyout. The list shows any rules, alerts, events, users, or hosts who's details you've viewed and can contain up to 10 unique entries. Click an entry in the list to quickly access the object's details.
```

Comment:
> The list shows any rules, alerts, events, users, or hosts who's details

@nastasha-solomon Overall looks good to me! I would suggest not list out all the types here, there is also IP flyout, and other teams are developing new flyouts that can show up in the history.